src/global/pipes/highlight.pipe.ts

```diff
@@ -4,28 +4,42 @@ import { ConfigService } from '../../app/config.service';
 
 @Pipe({ name: 'highlight' })
 export class HighlightPipe implements PipeTransform {
+  flag = 'g';
+  highlightStyle = 'style="font-weight: 600; background: #ffff00;"';
```

Comment:
Should be in the configuration file to be customized by the user

Two variables:
* font-weight
* background